rootex/framework/systems/render_system.cpp

```diff
@@ -102,9 +103,15 @@ void RenderSystem::popUIMatrix()
 
 void RenderSystem::setCamera(Ref<CameraVisualComponent> camera)
 {
+	m_Camera->setNotActive();
 	m_Camera = camera;
 }
 
+void RenderSystem::restoreCamera()
+{
+	m_Camera = Ref<CameraVisualComponent>(new CameraVisualComponent());
```

Comment:
Why create a new camera?
when no camera is set as the active camera, set a "default camera"